packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import { Image } from 'react-native';
+import { Formik } from 'formik';
+import {
+  ModalView,
+  Switch,
+  TextInput,
+  Button,
+  styled,
+  FlexedView,
+  PaddedView,
+} from '@apollosproject/ui-kit';
+import AddPrayerHeader from '../AddPrayerHeader';
+
+const StyledButton = styled(() => ({
+  borderRadius: 0,
+}))(Button);
+
+const BottomView = styled(() => ({
+  justifyContent: 'flex-end',
+}))(FlexedView);
+
+const SwitchContainer = styled(() => ({
+  width: '60%',
+  alignSelf: 'center',
+}))(PaddedView);
+
+const AddPrayerForm = memo(({ imgSrc, btnLabel, ...otherProps }) => (
+  <Formik
+    initialValues={{ prayer: '', anonymous: false }}
+    onSubmit={(values) => console.log(values)}
+  >
+    {({ handleChange, handleBlur, handleSubmit, values }) => (
+      <ModalView {...otherProps}>
+        <FlexedView>
+          <AddPrayerHeader imgSrc={imgSrc} />
+          {/* TODO: add underline={false} prop to TextInput, 
+        pending https://github.com/ApollosProject/apollos-prototype/issues/629 */}
+          <PaddedView>
+            <TextInput
```

Comment:
Is there anyway to make this TextInput more like a paragraph input rather than a single line input?

Reply:
Yep, totally. I think what makes it look like a single line is the underline. I have an open PR in core that gives us the option to style the `TextInput` without the underline with makes it look much more open.

Reply:
When I type into the input, it never returns to the next line but rather continues to travel horizontal. I am hoping we can create the input to function like the former.

Reply:
ah, I'll work on that